src/client/components/dashboard/dashboard_robot/view_model.ts

```diff
@@ -89,7 +90,7 @@ export class DashboardRobotViewModel {
       children: [
         Shape.of(
           MarkerGeometry.of({
-            heading: this.model.robotHeading.clone(),
+            heading: new Vector2(Math.cos(this.model.robotPosition.z), Math.sin(this.model.robotPosition.z)),
```

Comment:
Can you add this line here:
`// TODO (Annable): Add helper method for converting an angle to a unit vector.`